code/modules/research/xenobiology/crossbreeding/industrial.dm

```diff
@@ -166,7 +166,7 @@ Industrial extracts:
 
 /obj/item/slimecross/industrial/gold
 	colour = "gold"
-	effect_desc = "Produces random coins."
+	effect_desc = "Produces random coins. But no mythril, as it's to magical for something magical already."
```

Comment:
We probably don't need to clarify this--we don't give eldritch coins either.

Reply:
But the reasons are different
![image](https://user-images.githubusercontent.com/93882977/236759801-ca927df3-d2c7-402f-93eb-0c3501a29aed.png)